packages/strapi-plugin-upload/admin/src/containers/InputModalStepper/InputModalStepper.js

```diff
@@ -199,7 +200,13 @@ const InputModalStepper = ({ isOpen, onToggle, onInputMediaChange }) => {
       handleEditExistingFile(editedFile);
       goToList();
     } catch (err) {
-      console.log(err);
+      const errorMessage = get(
```

Comment:
You might need to improve the following to display the error from the serveur in case of a max size issue. The only thing is that you will have to handle the error when the user cancels the request. The following code should work:

```
 const errorMessage = get(
        err,
        ['response', 'payload', 'message', '0', 'messages', '0', 'message'],
        get(err, ['response', 'payload', 'message'], null)
      );

      if (errorMessage) {
        handleSetFileToEditError(errorMessage);
      }
```